homeassistant/components/qbittorrent/sensor.py

```diff
@@ -92,30 +89,77 @@ class QBittorrentSensor(SensorEntity):
 
     def __init__(
         self,
-        description: SensorEntityDescription,
         qbittorrent_client,
         client_name,
         exception,
+        server_unique_id,
+        description: SensorEntityDescription,
     ):
         """Initialize the qBittorrent sensor."""
         self.entity_description = description
         self.client = qbittorrent_client
         self._exception = exception
-
+        self._server_unique_id = server_unique_id
         self._attr_name = f"{client_name} {description.name}"
         self._attr_available = False
 
-    def update(self):
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return f"{self._attr_name}"
+
+    @property
+    def unique_id(self):
+        """Return the unique id of the sensor."""
+        return f"{self._server_unique_id}/{self._attr_name}"
+
+    @property
+    def state(self):
+        """Return the state of the sensor."""
+        return self._attr_native_value
+
+    @property
+    def available(self):
+        """Return true if device is available."""
+        return self._attr_available
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit of measurement of this entity, if any."""
+        return self.entity_description.native_unit_of_measurement
+
+    @property
+    def icon(self):
+        """Icon to use in the frontend, if any."""
+        return "mdi:cloud-download"
+
+    @property
+    def device_info(self):
+        """Return the device information of the entity."""
+        return {
+            "identifiers": {(DOMAIN, self._server_unique_id)},
+            "name": DOMAIN,
+            "model": DOMAIN,
+            "manufacturer": DOMAIN,
```

Comment:
DOMAIN isn't really end-user-friendly to display. Also maybe isn't doesn't suit the model either.